ansible/roles/spatial-service/templates/spatial-service-config.yml

```diff
@@ -201,22 +201,25 @@ security:
     check:
       serviceUrl:  {{api_key_service_url | default('https://auth.ala.org.au/apikey/ws/check?apikey=') }}
 
+# oidc related
   oidc:
     enabled: {{ oidc_enabled }}
+    clientId: {{ spatial_client_id | default(spatial_service_oidc_client_id) }}
```

Comment:
I moved this above the cognito conditional, because if not oidc cannot work without cognito.